forge-gui/res/cardsfolder/s/swinging_ship.txt

```diff
@@ -0,0 +1,12 @@
+Name:Swinging Ship
+ManaCost:no cost
+Types:Artifact Attraction
+Variant:A:Lights:2 6
+Variant:B:Lights:4 6
+K:Visit:TrigEffect
+SVar:TrigEffect:DB$ Effect | Triggers$ STShip | SpellDescription$ Visit — After the first combat phase this turn, there's an additional combat phase. At the beginning of that combat, untap all creatures that attacked this turn.
```

Comment:
why the extra effect?

Reply:
I've used Zariel, Archduke of Avernus as a template since that cards effect sounds most similar to this card. Would you impement it diffently ?

Reply:
it should probably be coded more like _World at War_
however the "first combat" part might be difficult then without more support
I'm just gonna approve it since taking Unfinity too seriously is against the spirit lol?